.devcontainer/nginx/pmm.conf

```diff
@@ -0,0 +1,220 @@
+  upstream managed-grpc {
```

Comment:
what is the difference between this file and one in pmm-server repository? It'll make us to handle two versions of nginx config and make development environment different from a real one.

Reply:
`auth_request` is an important part of the PMM development to be sure what works and what doesn't.
`/prometheus` should be alias for `/victoriametrics`

Reply:
it is needed for local development only, it can be used in docker-compose
```
    volumes:
      - .devcontainer/nginx/pmm.conf:/etc/nginx/conf.d/pmm.conf:ro
```

Reply:
and we have to support that, local environment should be as close as possible to production

Reply:
okay I can remove from PR and keep it local